src/OptimalTransport.jl

```diff
@@ -672,4 +675,162 @@ function quadreg(mu, nu, C, ϵ; θ=0.1, tol=1e-5, maxiter=50, κ=0.5, δ=1e-5)
     return sparse(γ')
 end
 
+## 1-Dimensional Optimal Transport Functions
+
+"""
+optimal_transport_cost(c,mu::ContinuousUnivariateDistribution,nu::UnivariateDistribution; plan=nothing)
+
+Calculates the optimal transport cost between μ to ν, where
+they are 1-Dimensional distributions and the cost
+function is of the form ``c(x,y) = h(|x-y|)`` such that
+``h`` is a convex function.
+
+Parameters:\n
+c: The cost function, which should be of the form ``c(x,y) = h(abs(x-y))``
+where `h` is a convex function.
+mu: 1-D distribution (e.g. `mu = Distributions.Normal(0,1)`)\n
+nu: 1-D distribution (e.g. `nu = Distributions.Normal(0,1)`)\n
+plan: Optional parameter in case the optimal transport plan is already known. Providing
+it can speed up calculations.
+"""
```

Comment:
I was avoiding the four spaces in the first line, because this is causing an error with Jupyter when rendering the docs. For some reason, if you add spaces in the first line, the math equations are not rendered when you visualize the documentation with jupyter.
But I guess this is an issue more with jupyter than with the docstring, so I'll use the standard.